### PR TITLE
Command Sender fix quotes

### PR DESCRIPTION
**Pull request by jmthomas**

### CODE REVIEW

File: openc3-init/plugins/packages/openc3-tool-cmdsender/src/tools/CommandSender/CommandSender.vue [hunk 1]
```diff
@@ -471,56 +470,61 @@ export default {
...
+          returnValue = String(value)
         }
       } else {
-        return_value = String(value)
+        returnValue = String(value)
+        // Ensure if this is a string that it is in quotes
+        // This is the case coming from the GUI but not from the cmd history
+        if (typeof value === 'string' && value.charAt(0) !== "'") {
+          returnValue = `'${returnValue}'`
+        }
```
Comment (jmthomas) [inline openc3-init/plugins/packages/openc3-tool-cmdsender/src/tools/CommandSender/CommandSender.vue:525]: Note this also quotes our states like `cmd("INST COLLECT with TYPE 'NORMAL', DURATION 1, OPCODE 171, TEMP 0")` but it was the only way to make ASCII commands like `cmd("INST ASCIICMD with STRING 'BLAH BLAH'")` work. Good news is if you manually delete the quotes it still works: `cmd("INST COLLECT with TYPE NORMAL, DURATION 1, OPCODE 171, TEMP 0")`

File: lib/cosmos/api/cmd_api.rb [hunk 2]
```diff
@@ -1,393 +0,0 @@
-module Cosmos
```
Comment (ryanmelt) [inline lib/cosmos/api/cmd_api.rb:1]: Where did cmd_api.rb go?

File: openc3-init/plugins/packages/openc3-tool-cmdsender/src/tools/CommandSender/CommandSender.vue [hunk 3]
```diff
@@ -681,18 +589,20 @@ export default {
       this.displaySendHazardous = false
       var paramList = this.createParamList()
 
+      // Ignore 500 errors because we might get a 500 error on out of range which we handle
+      localStorage.axiosIgnoreResponse = '500' // localStorage only supports strings
```
Comment (ryanmelt) [inline openc3-init/plugins/packages/openc3-tool-cmdsender/src/tools/CommandSender/CommandSender.vue:593]: We have to have a better way of doing this.  This is a global that will prevent 500 errors from this point forward, if for some reason it doesn't get unset.

**Review by ryanmelt — CHANGES_REQUESTED** [review]

File: lib/cosmos/api/cmd_api.rb (hunk 2 above)
Comment (jmthomas) [reply lib/cosmos/api/cmd_api.rb:1]: This was an accidental checkin to the root when I was doing some git diffing against older versions

File: openc3-init/plugins/packages/openc3-tool-cmdsender/src/tools/CommandSender/CommandSender.vue (hunk 3 above)
Comment (jmthomas) [reply openc3-init/plugins/packages/openc3-tool-cmdsender/src/tools/CommandSender/CommandSender.vue:593]: Agreed. Created https://github.com/OpenC3/openc3/issues/107 to address.

**Comment by ryanmelt** [comment]
Think you just need to fix the broken playwright tests now

**Pull request merged**